### PR TITLE
add logo for shared page to example theme

### PR DESCRIPTION
**Pull request by ennnnno**
Thanks for adding example theme [#16519](https://github.com/owncloud/core/pull/16519), nice! :+1:

I just dropped logo-wide for the shared page into example theme.

Please review @owncloud/designers @jancborchardt @karlitschek @schiesbn @pierrejochem

shared-page

### CODE REVIEW

File: themes/example/core/css/styles.css [hunk 1]
```diff
@@ -84,3 +84,8 @@ button.primary:focus,
 	width: 62px;
 	height: 34px;
 }
+#header .logo-wide {
+background-image: url("../img/logo-wide.svg");
+width: 150px;
+height: 34px;
```
Comment (LukasReschke) [inline themes/example/core/css/styles.css]: Indentation looks missing

**Comment by LukasReschke** [comment]
@owncloud-bot This is okay to test.

**Comment by LukasReschke** [comment]
> Thanks a lot for your contribution!
Contributions to the core repo require a signed contributors agreement http://owncloud.org/about/contributor-agreement/
> 
> Alternatively you can add a comment here where you state that this contribution is MIT licensed.

@ennnnno Can you please comment on this? THX.

**Comment by ennnnno** [comment]
of course it's open source MIT-Lizenz (c) 2015 [ennnnno](https://github.com/ennnnno)

**Comment by DeepDiver1975** [comment]
@jancborchardt

**Comment by jancborchardt** [comment]
@ennnnno actually this is not needed since we got rid of logo-wide in master. :)

**Comment by ennnnno** [comment]
@jancborchardt oh THX, of course! I've checked in stable

**Comment by MorrisJobke** [comment]
> @ennnnno actually this is not needed since we got rid of logo-wide in master. :)

So close this?

**Comment by jancborchardt** [comment]
@ennnnno you’re very welcome to contribute other design fixes though! :) Also join #owncloud-design on IRC

**Comment by ennnnno** [comment]
Tanks for pioneering and gathering :)

**Pull request closed without merging**